src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+            <td>
+                Parsing language                
+            </td>
+            <td>
+                You create grok patterns and regular expressions 
+            </td>
+            <td>
+                New Relic creates queries for you using the NRQL [aparse](/docs/nrql/nrql-syntax-clauses-functions/#func-aparse) function 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Timing
+            </td>
+            <td>
+                Applied at ingest
+            </td>
+            <td>
+                Applied when you query
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Results
+            </td>
+            <td>
+                Makes permanent changes to stored log 
+            </td>
+            <td>
+                Temporarily alters the logs table in the UI
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Live tail logs
+            </td>
+            <td>
+                Live tail logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Live tail logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Exported logs
+            </td>
+            <td>
+                Exported logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Exported logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Number of attributes
+            </td>
+            <td>
+                A maximum of [255 attributes](/docs/logs/log-api/log-event-data/#events) is available at ingest (the actual number attributes you can parse at ingest depends on the nature of your logs)
+            </td>
+            <td>
+                You can parse a maximum of 32 temporary attributes across all rules for query time parsing
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## How to create a query time parsing rule [#how-to] 
+
+Here's an overview of the tasks for creating query time parsing rules:
+
+<Steps>
+    <Step>
+        #### Select attribute value to parse [#select-attribute]
+        You can start creating a query time parsing rule by selecting an attribute to parse:
+
+            1. In the log table or in the **Formatted** tab of the log details view, highlight an attribute value you'd like to parse, known as the anchor string. Keep the following in mind:
+                * Your highlighted text should include the value(s) you want to extract and the surrounding string characters that will help identify the location of the extracted value(s). 
+                * If your initial anchor string is anywhere in the middle of the original attribute value, include at least one character before and after the values you want to extract.
```

Comment:
Can your SME's provide you with a sample? It would be nice to be able to see the anchor string you're talking about/see it broken up/in action

Reply:
@homelessbirds I'll see if I can add the example from the screenshot right into the procedure wording.